gui/additionsPane.py

```diff
@@ -79,9 +81,12 @@ def __init__(self, parent):
         self.gangPage = GangView(self.notebook)
         self.notebook.AddPage(self.gangPage, "Fleet", tabImage = gangImg, showClose = False)
 
+        self.notes = NotesView(self.notebook)
+        self.notebook.AddPage(self.notes, "Notes", tabImage = notesImg, showClose = False)
+
         self.notebook.SetSelection(0)
 
-    PANES = ["Drones", "Fighters", "Cargo", "Implants", "Boosters", "Projected", "Fleet"]
+    PANES = ["Drones", "Fighters", "Cargo", "Implants", "Boosters", "Projected", "Fleet", "Notes"]
```

Comment:
So this is a minor quibble, but it's a bit of a pet peeve of mine.  Because git blame goes line by line, if anyone ever adds/removed a single field from this, all the blame goes to that person.

Where I've come across this I've been changing it to:
```
PANES = [
    "Drones",
    "Fighters",
    "Cargo",
    "Implants",
    "Boosters",
    "Projected",
    "Fleet",
    "Notes",
]
```

You'll get git blamed for every line (but you were going to be anyway), and in the future adding a new tab won't assign blame for all tabs.  It also makes comparisons easier down the road.